### ISSUE TITLE
Bug: Chatwoot app not showing up new conversations 

### ISSUE THREAD

**Issue by nithindavid** [chatwoot/chatwoot-mobile-app#486]
**Describe the bug**

When I get notification on the mobile app and when I open the app. I am unable to see any conversations.

**Comment by nithindavid** [comment]
@muhsin-k Is this an existing bug?

**Comment by muhsin-k** [comment]
@nithindavid No. I just tested in the cloud version, and it is working fine. Anyone reported this issue?

**Comment by sojan-official** [comment]
@muhsin-k  is there a case when the logged in token is expired  ?

**Comment by sangroxx** [comment]
Hi! I am using the cloud version and facing the same issue.. Did reach out on Chatwoot's live chat, they said it's a bug with the mobile app.

**Comment by muhsin-k** [comment]
@sangroxx Could you please give more context about the issue. What error you are getting after clicking on notification?

**Comment by drexlma** [comment]
![0DE143E2-7F9A-48E3-A379-B60012C8BFC9](https://user-images.githubusercontent.com/7121202/164534337-995b1855-0808-462a-9451-0a518d446703.png)


This

**Comment by drexlma** [comment]
Logout & Relogin solve?

**Comment by sojan-official** [comment]
This should be fixed in https://github.com/chatwoot/chatwoot-mobile-app/issues/505
